app/models/classification.rb

```diff
@@ -487,8 +487,8 @@ def validate_uniqueness_on_tag_name
   end
 
   def validate_format_of_name
-    unless (name =~ /[^a-z0-9_:]/).nil?
-      errors.add("name", "must be lowercase alphanumeric characters, colons and underscores without spaces")
+    unless (name =~ /[^a-z0-9_:\-]/).nil?
```

Comment:
I'm wondering if this could have an impact on how we parse "tags" in MiqExpression - /cc @lpichler